addons/io/org.openhab.io.homekit/README.md

```diff
@@ -71,34 +65,34 @@ A full list of supported accessory types can be found in the table below.
  </tr>
  <tr>
   <td>&nbsp;</td>
-  <td>currentTemperature</td>
+  <td>CurrentTemperature</td>
   <td>Number</td>
-  <td>The current temperature, using the same unit rules as TemperatureSensor</td>
+  <td>The current temperature, same as above</td>
  </tr>
  <tr>
   <td>&nbsp;</td>
-  <td>heatingCoolingMode</td>
+  <td>homekit:HeatingCoolingMode</td>
```

Comment:
> The temperature format is very important to other integrations

You mean the unit? This will be addressed by https://github.com/eclipse/smarthome/issues/601.